src/deploy/functions/runtimes/node/index.ts

```diff
@@ -80,10 +92,77 @@ export class Delegate {
     return Promise.resolve(() => Promise.resolve());
   }
 
+  serve(
+    port: number,
+    adminPort: number,
```

Comment:
You can collapse port and adminPort. That's a prototype holdover that didn't scale to Python